packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -51,12 +50,9 @@ export class ColorPickerPopover<T extends CustomPickersDescriptor> extends React
   };
 
   handleChange = (color: any) => {
-    const { onColorChange, onChange, enableNamedColors, theme } = this.props;
+    const { onColorChange, onChange, theme } = this.props;
     const changeHandler = onColorChange || onChange;
 
-    if (enableNamedColors) {
-      return changeHandler(color);
```

Comment:
this should be the default

Reply:
okay

Reply:
@torkelo I can't recall when this has changed but it seems like the color picker popover always returns a hex color now, not the named color when the color from the palette is chosen. Is that something you have changed?

Reply:
@dprokop that can't be true, how did you test that? in master?

Reply:
testing in master I always get named colors when I click on the circle colors

Reply:
in the panel json

Reply:
My bad - I have not noticed this hasn't been updated in this PR yet. You are right - line 58 from the above code should be the default rather than `changeHandler(getColorForTheme(color, theme));`